src/OrbitGl/TriangleToggle.cpp

```diff
@@ -17,13 +17,15 @@
 #include "Track.h"
 
 TriangleToggle::TriangleToggle(State initial_state, StateChangeHandler handler,
-                               TimeGraph* time_graph, TimeGraphLayout* layout, Track* track)
+                               TimeGraph* time_graph, TimeGraphLayout* layout, Track* track,
+                               uint32_t indentation_level)
     : CaptureViewElement(track, time_graph, layout),
       track_(track),
       state_(initial_state),
       initial_state_(initial_state),
       handler_(std::move(handler)) {
-  SetSize(10.f, 10.f);
+  uint32_t capped_indentation_level = (indentation_level > 5) ? 5 : indentation_level;
```

Comment:
Why is there a cap at 5?

And I would write this as
```
constexpr uint32_t kMaxIndentationLevel = 5; 
uint32_t capped_indentation_level = std::min(indentation_level, kMaxIndentationLevel);
```

Reply:
Otherwise, the shape is simply to small. I added a comment.